android/src/toga_android/widgets/webview.py

```diff
@@ -23,14 +28,40 @@ def onReceiveValue(self, value):
         self.result.set_result(res)
 
 
+class TogaNavigationEvent:
+    def __init__(self, webresourcerequest):
+        self.request = webresourcerequest
+        self.cancel = False
+
+
+class TogaWebClient(static_proxy(WebViewClient)):
+    def __init__(self, impl):
+        super().__init__()
+        self.webview_impl = impl
+
+    @Override(jboolean, [A_WebView, WebResourceRequest])
+    def shouldOverrideUrlLoading(self, webview, webresourcerequest):
+        if self.webview_impl.interface.on_navigation_starting:
+            event = TogaNavigationEvent(webresourcerequest)
```

Comment:
I'm a little confused about what is going on here. TogaNavigationEvent is a Python class, wrapping a request and a cancel attribute. It's never assigned to anything, or passed as an argument to anything - then it's cleared. Why is it required at all?

Reply:
Yes, the TogaNavigationEvent was a left-over of some old code - I removed it.